nevergrad/benchmark/experiments.py

```diff
@@ -928,21 +928,32 @@ def control_problem(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     """MuJoCo testbed. Learn linear policy for different control problems.
     Budget 500, 1000, 3000, 5000."""
     seedg = create_seed_generator(seed)
-    funcs = [control.Ant(num_rollouts=5, random_state=seed),
-             control.Swimmer(num_rollouts=5, random_state=seed),
-             control.HalfCheetah(num_rollouts=5, random_state=seed),
-             control.Hopper(num_rollouts=5, random_state=seed),
-             control.Walker2d(num_rollouts=5, random_state=seed),
-             control.Humanoid(num_rollouts=5, random_state=seed)
+    num_rollouts = 1
+    funcs = [control.Swimmer(num_rollouts=num_rollouts, random_state=seed),
+             control.HalfCheetah(num_rollouts=num_rollouts, random_state=seed),
+             control.Hopper(num_rollouts=num_rollouts, random_state=seed),
+             control.Walker2d(num_rollouts=num_rollouts, random_state=seed),
+             control.Ant(num_rollouts=num_rollouts, random_state=seed),
+             control.Humanoid(num_rollouts=num_rollouts, random_state=seed)
              ]
-    optims = ["RandomSearch", "Shiwa", "CMA", "PSO", "OnePlusOne",
-              "NGOpt2", "DE", "Zero", "Powell", "Cobyla", "MetaTuneRecentering"]
 
-    for budget in [500, 1000, 3000, 5000]:
+    sigmas = [0.1, 0.1, 0.1, 0.1, 0.01, 0.001]
+    funcs2 = []
+    for i in range(6):
+        f = funcs[i].copy()
+        param: ng.p.Array = f.parametrization.copy()  # type: ignore
+        param.set_mutation(sigma=sigmas[i]).set_name(f"sigma={sigmas[i]}")
```

Comment:
always avoid indices when it's possible to iterate directly on the objects

```suggestion
    for sigma, func in zip(sigmas, funcs):
        f = func.copy()
        param: ng.p.Array = f.parametrization.copy()  # type: ignore
        param.set_mutation(sigma=sigma).set_name(f"sigma={sigma}")
```